pyproject.toml

```diff
@@ -1,3 +1,15 @@
+[tool.black]
+line-length = 120
+target-version = ['py37']
+include = '\.pyi?$'
+extend-exclude = '''
+^/(
+  | packages
+  | tools
```

Comment:
@nsoranzo I was actually going to ask about this -- I left this out thinking that even though Galaxy is 3.7+ some of the tools might be deployed using older pythons.  If that's not a concern, then I'm happy to drop that exclude.

Reply:
Mmmh, true, it may be that black changes the code in a 3.7+-only way.

Reply:
So that's exactly what I was worried about, but it occurs to me I could apply black to tools with a 3.6 profile, run again with 3.7 and see if there are any changes.  Sound good?  Then we can maybe drop the exclude.

Reply:
Sorry, I mean use black with the 3.6 profile (which I assume all tools are compatible with?), not the tool profile itself.